index.js

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env node
+require("dotenv").config();
+const axios = require("axios");
+
+// Usage from the command line:
+// npm install
+// node index sourceChannel targetChannel
+// e.g. node index general dev-chatter
+
+// Note: Do not include the hashtag in the channel name. Though I did account for stripping the hashtag away, process.argv returns undefined if the hashtag is present and I'm unsure of why.
+
+// Store Slack access token in a .env file in the root folder
+// You can ping a JDSB admin on Slack for the OAuth token, or request to become a collaborator on the app.
+// The token for an app can be found by selecting the app from api.slack.com/apps and navigating to "OAuth & Permissions" page after it has been installed to a workspace.
+const accessToken = process.env.SLACK_ACCESS_TOKEN;
+
+async function getConversationId(conversation) {
+
+  // Trim the pound sign if it was provided in the conversation argument
+  if (conversation.startsWith('#')) {
+    conversation = conversation.slice(1)
+  }
+
+  // Destructuring to pull out the channels result from the response
+  const { data: { channels } } = await axios.get(`https://slack.com/api/conversations.list?token=${accessToken}&pretty=1`)
+
+  // Find the channel with matching conversation name and pull out its ID
+  const conversationId = channels.find(chan => chan.name === conversation).id
```

Comment:
This is pulling out a channel id at the end from the conversations API. So, this method should in fact be named getChannelId().

Reply:
Can do -- believe that's what it was originally, until I switched over to the Conversation ID convention, and considered it could be used to migrate private convos down the line as well.

Reply:
@bhurstGH in the response, this is actually called a channel: https://api.slack.com/methods/conversations.list

So, I'm not sure what convention you're referring to.

Reply:
The *Conversation* API is the convention i was referring to. From your link:

> This Conversations API method returns a list of all channel-like conversations in a workspace. The "channels" returned depend on what the calling token has access to and the directives placed in the types parameter.

And the available types of conversations this could be used include public and private channels, as well as IM and "MPIM" -- which I guess is multi-person IM? Not sure.

No issue with changing it to channels in this context, but that was the logic: the use of the Conversation API and that the possible user lists could come from private messages in addition to channels.

Reply:
We'll make improvements for the Slack App migration.